### DISCUSSION TITLE
Core-data querying limit

### DISCUSSION THREAD

**Discussion by antiadominguez** [edgexfoundry/community#315] [Q&A]
I am doing a http request to core-data api to get readings of a resource in an specific interval. I want all the readings in that interval so I am putting the limit to -1 as the documentation said, to get all remaining readings. But I get in the response an incoherent number of readings compared to total count number. I also see that there isn't all the readings of the interval. Why is this issue happening?

**Comment by cloudxxx8** [comment]
You could try to modify the config of Service.MaxResultCount
https://github.com/edgexfoundry/edgex-go/blob/187954ff3d4f3dea9b0a451951d025e7f3bfc818/cmd/core-common-config-bootstrapper/res/configuration.yaml#L23

**Reply by antiadominguez** [reply]
I have changed it, downed the docker compose and turn it back to up and it's still returnning only 1024 readings

**Reply by cloudxxx8** [reply]
Your service may load the config from the config provider.

If you are using v3.1, please modify the config from Consul.
If you are using the main branch, you can add 
```
environment: 
  EDGEX_OVERWRITE_CONFIG: "true"
```
https://docs.edgexfoundry.org/4.0/microservices/configuration/CommonEnvironmentVariables/#edgex_overwrite_config

Or, try to clean all the volume and restart
`docker compose down -v`

**Reply by antiadominguez** [reply]
I am on v3.1, I have to change config on consul and then restart core-data. With that it returns more values, thank you
